kernel/kernel-pdfgenerator/pom.xml

```diff
@@ -75,7 +114,7 @@
 			<artifactId>bcpkix-jdk15on</artifactId>
 			<version>${bouncycastle.version}</version>
 		</dependency>
-		<dependency>
+		<!--<dependency>
```

Comment:
Let us remove the itext dependency block as it won't be used further.

Reply:
taken care